cmd/subctl/verify.go

```diff
@@ -149,21 +149,23 @@ func checkVerifyArguments(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("--connection-timeout must be >=20")
 	}
 
-	if _, _, err := getVerifyPatterns(verifyOnly, true); err != nil {
+	if _, _, err := getVerifySpecLabels(verifyOnly, true); err != nil {
 		return err
 	}
 
 	return checkNoArguments(cmd, args)
 }
 
-var verifyE2EPatterns = map[string]string{
-	component.Connectivity:     "\\[dataplane",
-	component.ServiceDiscovery: "\\[discovery",
-	"compliance":               "\\[compliance]",
+var verifyE2ESpecLabels = map[string]string{
```

Comment:
Can you update the commit message with an example on how basic tests can be executed?

Reply:
OK.  I also added a system test.